### DISCUSSION TITLE
[BP#61] Utiliser un thème léger

### DISCUSSION THREAD

**Discussion by Fred-artwai** [cnumr/best-practices#241] [Bonnes pratiques] [answered]
Est-ce que cette BP ne confond pas objectif et moyen ?
Utiliser un thème léger est un moyen pour atteindre la BP1,7,9 etc...

Je pourrais faire la même remarque pour la BP16

**Comment by ACTLEM** [comment]
J'ai du mal à comprendre la remarque. Ici, il s'agit de recommander certains choix de templates axés sur leur légèreté (donc une réflexion post conception fonctionnelle donc qui arrive après la BP 1). Par contre l'exemple donné n'est plus maintenu. Je changerais en prônant un design system comme celui du W3C (https://design-system.w3.org/) ou un système de template léger et plus "vivant".

**Reply by Fred-artwai** [reply]
Le thème Susty est vraiment extrême comme exemple. 
Déjà la notion de thème doit être bien défini et donc le titre doit indiquer qu'on se réfère à un CMS.
Mon impression c'est que cette BP est très subjective ou pas assez précise (j'ai moi même du mal à définir les critères objectifs d'un thème ecoconçu). Comment on peut orienter le lecteur de cette BP a faire le bon choix. On peut par exemple imaginer faire une interface minimaliste avec DIVI (horreur, malheur). Ou faire une interface assez fourni avec thème webperfé. Et encore une fois un thème ne suffit pas des fois il faut des composants en plus pour le rendre encore plus économe comme un plugin de cache, de minification, de concaténation etc... qui au final rendent le site assez propre.

Je sais pas si je suis assez claire ?

**Reply by ACTLEM** [reply]
Merci @Fred-artwai, c'est plus clair. Effectivement, cette bonne pratique est effectivement très subjective et est en soit un résultat attendu. J'ai du mal à trouver des arguments pour la garder. @rlemaire @tbroyer, des avis?

**Reply by tbroyer** [reply]
Ça ressemble à une application aux CMS de la BP#5 (voire même BP#6). +1 pour suppression (et/ou fusion dans BP#5)

**Reply by rlemaire** [reply]
+1 également.

**Comment by ACTLEM** [comment, marked as answer]
J'ai donc créé l'issue pour suppression: https://github.com/cnumr/best-practices/issues/277
